apps/frontend/src/app/poll/choose-events/choose-events.component.ts

```diff
@@ -23,16 +22,19 @@ export class ChooseEventsComponent implements OnInit {
   isAdmin: boolean = false;
 
   // aux
-  checks: CheckboxState[] = [];
-  editChecks: CheckboxState[] = [];
   bookedEvents: boolean[] = [];
   bestOption: number = 1;
   closedReason?: string;
   showResults = false;
 
   // view state
-  name: string = '';
+  newParticipant: CreateParticipantDto = {
+    name: '',
+    selection: {},
+    token: '',
+  };
```

Comment:
This could be moved into the `clearSelection`-function.
It could then be renamed to something like `resetNewParticipant()` (hopefully you have a better name for that than I have).

Reply:
Not possible, because the field needs to be initialized.